project-base/app/tests/FrontendApiBundle/Functional/Order/GetOrderItemsTest.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\FrontendApiBundle\Functional\Order;
+
+use App\Model\Order\Item\OrderItem;
+use App\Model\Order\Order;
+use DateTime;
+use DateTimeInterface;
+use PHPUnit\Framework\Attributes\DataProvider;
+use Shopsys\FrameworkBundle\Model\Order\Item\OrderItemTypeEnum;
+use Shopsys\FrameworkBundle\Model\Order\Status\OrderStatusTypeEnum;
+use Tests\FrontendApiBundle\Test\GraphQlWithLoginTestCase;
+
+class GetOrderItemsTest extends GraphQlWithLoginTestCase
+{
+    /**
+     * @param array $queryVariables
+     * @param array $expectedOrderItemsIds
```

Comment:
```suggestion
     * @param int[] $expectedOrderItemsIds
```

Not sure, how to make the `$queryVariables` annotation more specific, though :smile:

Reply:
[fixup! order items query test](https://github.com/shopsys/shopsys/pull/3295/commits/392f3fea5ebe0e1e3ef6650959d80da389b14f77)